charts/datadog/templates/_container-security-agent.yaml

```
@@ -34,9 +34,11 @@
     {{- if .Values.datadog.securityAgent.compliance.enabled }}
     - name: DD_COMPLIANCE_CONFIG_CHECK_INTERVAL
       value: {{ .Values.datadog.securityAgent.compliance.checkInterval | quote }}
-    {{- if .Values.datadog.securityAgent.compliance.xccdf.enabled }}
+    {{- if .Values.datadog.securityAgent.compliance.xccdf.enabled || .Values.datadog.securityAgent.compliance.host_benchmarks.enabled }}
```

My bad, my suggestion in a previous comment was incorrect.

```suggestion
    {{- if or .Values.datadog.securityAgent.compliance.xccdf.enabled .Values.datadog.securityAgent.compliance.host_benchmarks.enabled }}
```